src/main/java/ch/njol/skript/effects/EffPush.java

```diff
@@ -77,6 +77,10 @@ protected void execute(final Event e) {
 			final Vector mod = d.getDirection(en);
 			if (v != null)
 				mod.normalize().multiply(v.doubleValue());
+			if (!Double.isFinite(mod.getX()) || !Double.isFinite(mod.getY()) || !Double.isFinite(mod.getZ())) {
```

Comment:
```suggestion
			if (!(Double.isFinite(mod.getX()) && Double.isFinite(mod.getY()) && Double.isFinite(mod.getZ()))) {
```

Reply:
Well done for spotting, this is the more efficient way to check it. :)